packages/components/src/FormatFile/FormatFile.mdx

```diff
@@ -0,0 +1,57 @@
+---
+name: FormatFile
+menu: Components
+route: /components/format-file
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { FormatFile } from ".";
+import { Content } from "../Content/";
+
+# File Card
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Pass this a logical description of a file (an object) and it will represent a
+file in a number of lifecycle states.
+
+A progress indicator can be used to display upload status, and a thumbnail or
+icon will be displayed depending on what is provided.
+
+A delete callback function can be passed in, and will be called when the user
+actions the delete button.
+
+```ts
+import { FormatFile } from "@jobber/components/FormatFile";
+```
+
+<Playground>
+  <Content>
+    <FormatFile
+      file={{
+        key: "abc",
+        name: "myballisbigandroundIamrollingitontheground.png",
+        type: "image/png",
+        size: 213402324,
+        progress: 0.3,
+        src: () => Promise.resolve("https://source.unsplash.com/250x250")
+      }}
+      onDelete={() => alert("🧨BALEETED!💥")}
+    />
+    <FormatFile
+      file={{
+        key: "mosdef",
+        name: "tps_report.png",
+        type: "application/pdf",
+        size: 2134023,
+        progress: 1
+      }}
+      onDelete={() => alert("🧨BALEETED!💥")}
+    />
+  </Content>
+</Playground>
+
+## Props
```

Comment:
@JeffMarvin / @rebecca-li Any Design & Usage Guidelines you think would be good to add in here?

https://github.com/GetJobber/atlantis/blob/master/packages/generators/templates/component/%7B%7Bname%7D%7D.%7B%7Bmdx%7D%7D#L30

Reply:
- FormatFile components should take up the full width of the parent container (1 or 2 files per row)